server/config/dev.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   mongoURI: process.env.MONGO_DEV,
   secretKey: process.env.SECRET_KEY,
+  queryPageSize: 100,
```

Comment:
@dzimmerman902 is there a place to store application-wide configuration settings like this? Not sure if this is the right place or not.